### ISSUE TITLE
Version 1.4.1 undefined local variable or method `controller'

### ISSUE THREAD

**Issue by kntmrkm** [JangoSteve/remotipart#195]
After update to `1.4.1`.

```
 module RequestHelper
    def remotipart_submitted?
      controller && params[:remotipart_submitted] ? true : false
    end
    
    alias :remotipart_requested? :remotipart_submitted?
```

My env?

**Comment by mshibuya** [comment]
Hmm... 😰 
Can you reproduce it using this?
https://github.com/kntmrkm/rails51/tree/remotipart

**Comment by mshibuya** [comment]
Pushed another fix as eece683d5dfb8b9d0344a706cb4cdfec4248098e, please see if it works and not breaks web-console.

**Comment by kntmrkm** [comment]
https://github.com/JangoSteve/remotipart/commit/eece683d5dfb8b9d0344a706cb4cdfec4248098e

It work. and not break web-console.

**Comment by mshibuya** [comment]
Thanks for the help, just published 1.4.2.
